.gitignore

```diff
@@ -26,3 +26,6 @@ venv/
 
 # Setuptools scm version
 _version.py
+
+# mkdocs documentation
+/site
```

Comment:
I think we want to remove the leading `/` since I'm _guessing_ this absolute path is only correct if developing inside a docker container

Reply:
I am sorry, but I don't understand which docker container you are mentioning.

I do not use docker container for my development environment. I simply followed [CONTRIBUTION.md](https://github.com/IBM/oper8/blob/main/CONTRIBUTING.md#development-environment-setup), and it does not have any description for docker.

And when I build the document with `mkdocs build` command, it outputs the document into `/site` folder, so I updated `gitignore` to ignore this directory.

Is there a way to create the development environment with docker?